app/javascript/components/pages/Jobs.jsx

```diff
@@ -11,15 +12,20 @@ import 'stylesheets/jobs';
 
 const Jobs = () => {
     const [jobs, setJobs] = useState([]);
+    const [cookies] = useCookies('token');
 
     useEffect(() => {
         const fetchData = async () => {
-            const data = await getJobs();
-            setJobs(data);
+            try {
+                const data = await getJobs(cookies['token']);
+                setJobs(data);
+            } catch (error) {
```

Comment:
what do you think about redirecting only in the case of the unauthorized error? I have an inkling that this might swallow other types of unexpected bugs in a weird way!